remo/base/views.py

```diff
@@ -38,14 +39,23 @@ def login_failure(self, msg=''):
         return super(OIDCCallbackView, self).login_failure()
 
 
-@cache_control(private=True, no_cache=True)
+@cache_control(private=True, max_age=60 * 60 * 8)
```

Comment:
`cache_page` will cache the view’s response causing the same problems mentioned in the first comment. We should cache only the response from the planet and not the view.